src/pages/wasmd/getting-started/cli.mdx

```diff
@@ -557,6 +558,62 @@ The output will be similar to the following:
 }
 ```
 
+### Query Contract State Raw
+
+To query the state of a WASM contract for a specific key, you can use the following command:
+
+```sh
+# Set the state key you want to query
+KEY="636F6E666967"
+wasmd q wasm contract-state raw  $CONTRACT $KEY -o json
+```
+
+The output will look similar to this:
+
+```json
+{
+  "data": "eyJ2ZXJpZmllciI6Indhc20xNzlhdnc5NmFheTcwcHM5OXVtdWFlc3h4Y3p3YzBxbTVnd3VmeGciLCJiZW5lZmljaWFyeSI6Indhc20xNDI3a3BxOW1tbmZwMG1hZGs1YXhoMnVrbWpncGZoNnNremR4a3UiLCJmdW5kZXIiOiJ3YXNtMTc5YXZ3OTZhYXk3MHBzOTl1bXVhZXN4eGN6d2MwcW01Z3d1ZnhnIn0="
+}
+```
+
+`"data"` represents the base64-encoded state data of the contract for the given key.
+
+You can decode the `data` field using the following command:
+
+```sh
+wasmd q wasm contract-state raw  $CONTRACT $KEY -o json | jq -r '.data' | base64 -d
+```
+
+The output will be similar to the following:
+
+```json
+{
+  "verifier": "wasm1hvgm6p76gccgg4dl4caa8a7v03dsqww6r9sk4g",
+  "beneficiary": "wasm1pa29lac5s85kgj7pn9z6gc0t4sqgzllcguhf24",
+  "funder": "wasm1hvgm6p76gccgg4dl4caa8a7v03dsqww6r9sk4g"
+}
+```
+
+### Query Contract State Smart
+
+To query the state of a WASM contract for specific data, you can use the following command:
```

Comment:
I'd add a line here that the content of `QUERY` depends on the specific contract you want to call and add a link to `core/entrypoints/query`